A79-non-per-call-metrics-architecture.md

```diff
@@ -429,8 +434,8 @@ public interface MetricsRecorder {
    * @param labelValues  Required labels for identifying the metric.
    * @param optionalLabelValues Additional labels to provide more context.
    */
-  default void recordDoubleHistogram(MetricDescriptor histogramDescriptor,
-      Double value, List<String> labelValues, List<String> optionalLabelValues)
+  default void recordDoubleHistogram(DoubleHistogramDescriptor histogramDescriptor, 
```

Comment:
can you use mdformat on this doc please? There are spaces at the end of the line

Reply:
Thanks for letting me know. Fixed